src/Tests/SettingsDataTrait.php

```diff
@@ -199,6 +199,7 @@ private function getValidVisibilitySettings() {
    */
   private function getValidAdvancedSettings() {
     return [
+      'bootstrap_mode' => 'auto',
```

Comment:
Let's build the test based on non-default value, in this case let's change the value from 'auto' to 'manual'.